src/main/java/controller/GameController.java

```diff
@@ -0,0 +1,81 @@
+package controller;
+
+import racingcar.Car;
+import racingcar.CarRepository;
+import racingcar.RacingCarGame;
+import view.InputView;
+import view.OutputView;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static constant.Constants.*;
+
+public class GameController {
+    private static final int MINIMUM_NUMBER_OF_MOVING = 1;
+    private static final String CAR_NAME_DELIMITER = ",";
+
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final RacingCarGame racingCarGame;
+
+    public GameController(OutputView outputView) {
+        this.inputView = new InputView(outputView);
+        this.outputView = outputView;
+        this.racingCarGame = new RacingCarGame();
+    }
+
+    public void run() {
+        addCars(inputView.readCarNames());
+        startRacing(readNumberOfMoving());
+        outputView.printWinners(racingCarGame.getWinners());
+    }
+
+    private void addCars(String carNames) {
+        try {
+            CarRepository.updateCars(makeCars(carNames));
+        } catch (IllegalArgumentException illegalArgumentException) {
+            outputView.printException(illegalArgumentException.getMessage());
+            addCars(inputView.readCarNames());
+        }
+    }
+
+    private List<Car> makeCars(String carNames) {
+        return Arrays.stream(carNames.split(CAR_NAME_DELIMITER))
+                .filter(this::validateCarName)
+                .map(Car::new)
+                .collect(Collectors.toList());
+    }
+
+    private boolean validateCarName(String carName) {
+        if (carName.isEmpty()) {
+            throw new IllegalArgumentException(COMMA_NOT_ALLOWED_EXCEPTION);
+        }
+        return true;
+    }
+
+    private int readNumberOfMoving() {
+        try {
+            int numberOfMoving = inputView.readNumberOfMoving();
+            validateTrial(numberOfMoving);
+            return numberOfMoving;
+        } catch (IllegalArgumentException exception) {
+            outputView.printException(exception.getMessage());
+            return readNumberOfMoving();
+        }
+    }
+
+    private void validateTrial(int numberOfMoving) {
+        if (numberOfMoving < MINIMUM_NUMBER_OF_MOVING) {
+            throw new IllegalArgumentException(TRIAL_INPUT_EXCEPTION);
+        }
+    }
+
+    private void startRacing(int numberOfMoving) {
+        for (int round = 0; round < numberOfMoving; round++) {
+            racingCarGame.repeatRounds();
+            outputView.printCarsStatus(CarRepository.findAll());
+        }
```

Comment:
round 인스턴스가 반복문내에 사용되지 않고 있네요 :) 

```suggestion
    private void startRacing(int numberOfMoving) {
        while (numberOfMoving -- > 0) {
            racingCarGame.repeatRounds();
            outputView.printCarsStatus(CarRepository.findAll());
        }
```
은 어떤가요?

Reply:
사실 가독성 측면에서 위의 for문이 더 낫지 않을까?? 생각했었는데, 리뷰어님 피드백 보고 for문의 round 변수가 아예 사용되지 않고 있다는 부분을 처음 보게 되었네요. 그래서 리뷰어님이 적어주신 while 문으로 리팩토링 진행했습니다.